training_pipeline/pipeline/pipeline.py

```diff
@@ -101,6 +105,7 @@ def create_pipeline(
     )
     components.append(evaluator)
```

Comment:
since we are training for very few epochs(1 or 2), sometimes the model doesn't pass the baseline threshold. If that happens, any pushers on the downstream will be ignored.

in order to evaluate if the HFPusher works fine, we could directly connect the `Trainer` to `HFPuser`. Then, if everything is fine, we could switch back.

Reply:
In the latest commit (https://github.com/deep-diver/semantic-segmentation-ml-pipeline/pull/34/commits/13ee2f9210e51de47d35b110d6ff758e10fbf545), I upped the number of epochs to 5. I triggered the pipeline too: https://github.com/deep-diver/semantic-segmentation-ml-pipeline/actions/runs/3303974276/jobs/5452538109.